changelog.txt

```diff
@@ -1,3 +1,6 @@
+0.82.0-SNAPSHOT
```

Comment:
Remove SNAPSHOT. Changelog is only for "official releases". I meant that the file `version.txt` should be updated to `0.82.0-SNAPSHOT` as well as `README.md`. This is because SNAPSHOT releases are deployed automatically when committed to master and since this a new feature it require a minor bump from 0.81.1 to 0.82.0 :)